library/src/blas2/rocblas_hpr2.hpp

```diff
@@ -0,0 +1,150 @@
+/* ************************************************************************
+ * Copyright 2016-2020 Advanced Micro Devices, Inc.
+ * ************************************************************************ */
+#pragma once
+#include "handle.h"
+#include "rocblas.h"
+
+template <typename T, typename U>
```

Comment:
Is U needed? Is 'T alpha' a problem here?

Reply:
Nope, alpha should be able to be a T, this must have been left from hpr. Will fix.